pkgs/development/python-modules/applicationinsights/default.nix

```diff
@@ -0,0 +1,23 @@
+{ buildPythonPackage
+, portalocker
+}:
+
+buildPythonPackage rec {
+  version = "0.11.9";
+  pname = "applicationinsights";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1hyjdv6xnswgqvip8y164piwfach9hjkbp7vc2qzhd7amjpim89h";
+  };
+
+  propagatedBuildInputs = [ portalocker ];
+
+  meta = with stdenv.lib; {
+    description = "This project extends the Application Insights API surface to support Python.";
+    homepage = https://github.com/Microsoft/ApplicationInsights-Python;
+    license = licenses.mit;
+    maintainers = with maintainers; [ jonringer ];
+    platforms = platforms.unix;
```

Comment:
```suggestion
```

Don't see why we'd need to set platforms when `buildPythonPackage` already does this.
Was there an intention to restrict it?

Reply:
Don't remember the original reasoning, ill remove it.